Hometask_1/Hometask_1.py

```diff
@@ -0,0 +1,29 @@
+import random  # importing random module
+
+#  Getting 100 numbers from 0 to 1000
+hundred_random_numbers = []  # Create a list
+for _ in range(100):  # Define count of numbers in the list
+    hundred_random_numbers.append(random.randint(0, 1000))  # Create the list of numbers from 0 to 1000
+# Sort list from min to max
+sorted_numbers = []  # Create a new list for further sorted numbers from min to max
+while hundred_random_numbers:
+    minimum = hundred_random_numbers[0]  # Put min value on the beginning of the list
```

Comment:
Please, correct your comment. It's confusing.
![image](https://user-images.githubusercontent.com/83510806/218543092-a2506c31-087e-464f-8026-77a9481ade90.png)